### PR TITLE
First fix for https://github.com/OWASP/owasp-masvs/issues/302

### PR DESCRIPTION
**Pull request by commjoen**
Thank you for submitting a Pull Request to the Mobile Security Testing Guide. Please make sure that:

- [x] Your contribution is written in the 2nd person (e.g. you)
- [x] Your contribution is written in an active present form for as much as possible.
- [x] You have made sure that the reference section is up to date (e.g. please add sources you have used, make sure that the references to MITRE/MASVS/etc. are up to date)
- [x] Your contribution has proper formatted markdown and/or code
- [x] Any references to website have been formatted as [TEXT](URL “NAME”)
- [x] You verified/tested the effectiveness of your contribution (e.g.: is the code really an effective remediation? Please verify it works!)

If your PR is related to an issue. Please end your PR test with the following line:
This PR covers issue https://github.com/OWASP/owasp-masvs/issues/302.

### CODE REVIEW

**Comment by OMTGreleaser** [comment]
Results for commit 9ecfc168f80093f3c8f83935a558073bbefb594d: Broken link result: [!] Error(s) found by the Links validator: 10 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        0 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: Document/0x04i-Testing-user-interaction.md [hunk 1]
```diff
@@ -30,6 +32,10 @@ Here is a list of best practices where a user could be informed of:
 - **Specific credentials**: When a user gets a recovery code, a password or a pin from the application (or sets one), instruct the user to never share this with anyone else and that only the app will request it.
 - **Application distribution**: In case of a high-risk application it is recommended to communicate what the official way of distributing the app is. Otherwise, users might use other channels in which they download a compromised version of the application.
 
+#### Access to device-data
+
+Allthough partially covered by the Google Play Store and the Apple App Store, you still need to explain to the user which services your app consumes and why. For instance: does your app require access to the contact list? Explain the user why. Does your app need access to location services of the device? Explain the user why. Does your app use device-identifiers to identify the device? Explain the user why your app needs to do that.
```
Comment (sushi2k) [inline Document/0x04i-Testing-user-interaction.md]: Maybe add something like: 

Good examples for proper explanations can be found in the: 
- [Apple Human Interface Guidelines](https://developer.apple.com/design/human-interface-guidelines/ios/app-architecture/requesting-permission/ "Apple Human Interface Guidelines") and
- in the [Android App permissions best practices](https://developer.android.com/training/permissions/requesting.html#explain "Android App permissions best practices").

File: Document/0x04i-Testing-user-interaction.md [hunk 2]
```diff
@@ -5,7 +5,9 @@
 A lot has happened lately in terms of responsibilities that developers have to educate users on what they need to know.
 This has shifted especially with the introduction of the [General Data Protection Regulation (GDPR)](https://gdpr-info.eu/ "GDPR") in Europe. Ever since then, it is best to educate users on what is happening with their private data and why.
 Additionally, it is a good practice to inform the user about how he can best use the application to ensure a secure processing of his information.
-Both items will be dealt with here.
+Next, a user should be informed on what type of device-data the app will access, whether that is PII or not.
```
Comment (cpholguera) [inline Document/0x04i-Testing-user-interaction.md]:
```suggestion
Next, a user should be informed on what type of device data the app will access, whether that is PII or not.
```

File: Document/0x04i-Testing-user-interaction.md (hunk 1 above)
Comment (cpholguera) [inline Document/0x04i-Testing-user-interaction.md]:
```suggestion
Although partially covered by the Google Play Store and the Apple App Store, you still need to explain to the user which services your app consumes and why. For instance:

- Does your app require access to the contact list?
- Does your app need access to location services of the device?
- Does your app use device identifiers to identify the device?

Explain the user why your app needs to do this kind of things.
```

Comment (cpholguera) [reply Document/0x04i-Testing-user-interaction.md]: made it a bit more fluid with the list and removed duplicated sentences.

File: Document/0x04i-Testing-user-interaction.md [hunk 3]
```diff
@@ -30,6 +32,10 @@ Here is a list of best practices where a user could be informed of:
 - **Specific credentials**: When a user gets a recovery code, a password or a pin from the application (or sets one), instruct the user to never share this with anyone else and that only the app will request it.
 - **Application distribution**: In case of a high-risk application it is recommended to communicate what the official way of distributing the app is. Otherwise, users might use other channels in which they download a compromised version of the application.
 
+#### Access to device-data
```
Comment (cpholguera) [inline Document/0x04i-Testing-user-interaction.md]: Style guide: Chicago style
```suggestion
#### Access to Device Data
```

File: Document/0x04i-Testing-user-interaction.md [hunk 4]
```diff
@@ -30,6 +32,10 @@ Here is a list of best practices where a user could be informed of:
...
+
+Allthough partially covered by the Google Play Store and the Apple App Store, you still need to explain to the user which services your app consumes and why. For instance: does your app require access to the contact list? Explain the user why. Does your app need access to location services of the device? Explain the user why. Does your app use device-identifiers to identify the device? Explain the user why your app needs to do that.
+
 #### Other information you have to share (OSS information)
```
Comment (cpholguera) [inline Document/0x04i-Testing-user-interaction.md]: Style guide: Chicago style
```suggestion
#### Other Information You Have to Share (OSS Information)
```

**Comment by OMTGreleaser** [comment]
Results for commit 43544d1150c95e0d7e70a92512e19dfe0d274059: Broken link result: [!] Error(s) found by the Links validator: 8 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        0 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: Document/0x04i-Testing-user-interaction.md [hunk 5]
```diff
@@ -47,3 +59,8 @@ Given copyright laws, you must make sure you inform the user on any third party
 #### Website to help with understanding licenses
 
 - <https://tldrlegal.com/>
+
+#### Guidance on permission requesting
```
Comment (cpholguera) [inline Document/0x04i-Testing-user-interaction.md]:
```suggestion
#### Guidance on Permission Requesting
```

File: Document/0x04i-Testing-user-interaction.md [hunk 6]
```diff
@@ -47,3 +59,8 @@ Given copyright laws, you must make sure you inform the user on any third party
 #### Website to help with understanding licenses
```
Comment (cpholguera) [inline Document/0x04i-Testing-user-interaction.md]:
```suggestion
#### Website to Help with Understanding Licenses
```

**Comment by OMTGreleaser** [comment]
Results for commit 4f65ae2fb10433ee23b34c6d4bdbd72723add2d9: Broken link result: [!] Error(s) found by the Links validator: 6 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        0 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by commjoen** [comment]
Thanks guys!

**Pull request merged**